golang/.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
this seems like a "your specific IDE" kind of thing and doesn't really have much to do with the project itself. would it make more sense to remove this and just not commit the IntelliJ project folder?

Reply:
Well, in fact, IntelliJ will always create this folder in the project root so, if I don't add this, anyone could forget to remove the folder when pushing. So, I prefer this here instead of removing the folder every time someone forgets to do it.

Reply:
Yeah, I think `.idea` should be included in the `.gitignore` 👍